### PR TITLE
Create DB from diagram along with tables

### PR DESCRIPTION
**Pull request by yonas44**
Hello👋, for this PR I worked solo to implement the requirements since my coding partner wasn't available to collaborate on the project, after talking with student success I was allowed to work on the projects solo, and as proof of that here is a screenshot of the email from @Maggie from student_success.
solo
So as per the requirement for the project:
- Create a database (name: optional) `clinic`
- Create tables `medical_histories`, `treatments`, `patients`, `invoices`, `invoice_items` and a join table called `medical_treatment_history` from medical_histories and treatment tables.

### Thank you for reviewing my PR!

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	id INT GENERATED ALWAYS As IDENTITY,
+	name VARCHAR,
+	date_of_birth DATE,
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_treatment_histories (
+    md_history_id INT REFERENCES medical_histories(id),
+    treatment_id INT REFERENCES treatments(id)
+);
```
Comment (BertrandConxy) [inline schema_based_on_diagram.sql]: - [x] Good job so far 👏 👏 However, I can see that you have not added index on the foreign keys in various tables while it is among the project requirements. Apart from that, index helps to run queries very fast so as you understand, they are important. Please add them too 😃 👍 

**Requirement**
image

**HINT 💡**
```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
...
```

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @yonas44, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (yonas44) [reply schema_based_on_diagram.sql]: Thank you @BertrandConxy, I will add them🙏

**Review by rustam-tolipov — APPROVED** [review]
Hi @yonas44, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**